src/test/ml-modules/root/test/suites/auditing/remerge.sjs

```diff
@@ -29,7 +29,7 @@ function findMergedDoc(uris) {
 const doc12URI = findMergedDoc([lib.URI1, lib.URI2]);
 
 xdmp.invokeFunction(
-  function() { merging.saveMergeModelsByUri([lib.URI3, lib.URI4], merging.getOptions(lib['OPTIONS-NAME'])) },
+  function() { merging.saveMergeModelsByUri([lib.URI3, lib.URI4], merging.getOptions(lib['OPTIONS-NAME'], con['FORMAT-XML'])) },
```

Comment:
I'm pretty sure you can do this javascript style. 

`con.FORMATXML`

It just removes the minus sign when making keys available.

Reply:
changed